### PR TITLE
add application endpoints

### PR DESCRIPTION
**Pull request by anton-barabash**

### CODE REVIEW

File: src/ApplicationList/function.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        "get"
+      ]
+    },
+    {
+      "type": "http",
+      "direction": "out",
+      "name": "$return"
+    },
+    {
+      "type": "table",
```
Comment (camiloramirezf) [inline src/ApplicationList/function.json]: Why do we need a binding here to InternalTranslations?

File: src/Shared/azureStorage/types.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { TableService } from '../../../node_modules/azure-storage';
```
Comment (camiloramirezf) [inline src/Shared/azureStorage/types.ts]: I believe this import can be simplified to import { TableService } from 'azure-storage'?

Comment (anton-barabash) [reply src/Shared/azureStorage/types.ts]: Good catch. Fixed.

File: src/ApplicationList/function.json (hunk 1 above)
Comment (anton-barabash) [reply src/ApplicationList/function.json]: I'll refactor this part later, once I read the documentation related to function files.

**Pull request merged**